src/Files.App/UserControls/SidebarControl.xaml.cs

```diff
@@ -140,8 +137,8 @@ public ICommand EmptyRecycleBinCommand
 
 		public SidebarControl()
 		{
-			this.InitializeComponent();
-			this.Loaded += SidebarNavView_Loaded;
+			InitializeComponent();
+			Loaded += SidebarNavView_Loaded;
```

Comment:
Can this be registered from the xaml side? I think it's preferred to register events that way.

Reply:
Done.